image/image.go

```diff
@@ -49,6 +49,13 @@ var validRefMediaTypes = []string{
 }
 
 func validate(w walker, refs []string, out *log.Logger) error {
+	var tmperr error
```

Comment:
I agree with you, but I updated it [here](https://github.com/opencontainers/image-tools/blob/0a1e5dc0c9b67619d46bd8cbcc948bda1b143b27/image/image.go#L68), because `ref` is used at [here](https://github.com/opencontainers/image-tools/blob/0a1e5dc0c9b67619d46bd8cbcc948bda1b143b27/image/image.go#L69)